stock_cycle_count/models/stock_inventory.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class StockInventory(models.Model):
+    _inherit = 'stock.inventory'
+
+    @api.one
+    def _compute_inventory_accuracy(self):
+        total_qty = sum(self.line_ids.mapped('theoretical_qty'))
+        abs_discrepancy = sum(self.line_ids.mapped(
+            lambda x: abs(x.discrepancy_qty)))
+        if total_qty:
+            self.inventory_accuracy = 100 * (total_qty - abs_discrepancy) / \
```

Comment:
Shouldn't be 100.0 ?
And this one shouldn't be as static variable (I don't like hardcoded values... subjective) ?